src/witan/models/dem/ccm/mort/mortality.clj

```diff
@@ -55,75 +55,106 @@
        (calc-death-rates historic-deaths)
        (hash-map :historic-asmr)))
 
-(defworkflowfn project-asmr-average-fixed
-  "Takes a dataset with historic mortality rates, and parameters for
+(defworkflowfn project-asmr-1-1-0
+  "May take a dataset with historic mortality rates, and parameters for
   the number of years of data to average for the calculation, and the
   jumpoff year. Returns a dataset that now includes projected
   mortality rates, projected with the jumpoff year average method (see
-  docs)"
-  {:witan/name :ccm-mort/project-asmr
-   :witan/version "1.0.0"
-   :witan/input-schema {:historic-asmr HistASMRSchema}
-   :witan/param-schema {:start-yr-avg-mort s/Int :end-yr-avg-mort s/Int}
-   :witan/output-schema {:initial-projected-mortality-rates ProjFixedASMRSchema}
-   :witan/exported? true}
-  [{:keys [historic-asmr]} {:keys [start-yr-avg-mort end-yr-avg-mort]}]
-  {:initial-projected-mortality-rates (cf/jumpoffyr-method-average historic-asmr
-                                                                   :death-rate
-                                                                   :death-rate
-                                                                   start-yr-avg-mort
-                                                                   end-yr-avg-mort)})
-
-(defworkflowfn project-asmr-average-applynationaltrend
-  "Takes a dataset with projected death rates for the future, a dataset with
+  docs)
+  Or
+  Takes a dataset with projected death rates for the future, a dataset with
   historic mortality rates, and parameters for: first projection year, last
   projection year, the scenario to use in the future rates dataset (e.g. :low,
   :principal or :high), the start and end  years of data to average for the calculation,
   and the last year of mortality data. Returns a dataset that now includes projected
   mortality rates, projected with the jumpoff year average method combined with an
   applied national trend (see docs)"
```

Comment:
"Takes a back series of age-specific mortality rates and a start and end year on which to base calculation of the projected mortality rates in the first loop projection year from the historic mortality rates (method dependent on average or trend jump off year method variant). For projecting mortality rates in years following the first projection year up until the last projection year: the fixed method applies the jump off year rates each year; the apply national trend method requires a national trend dataset and a scenario to use as the future mortality trend assumption, and generates variable rates for each year. Outputs a dataset of projected age-specific mortality rates for each projection year"